src/page/ConfigPageLayout/tabs/GeneralTab.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { config } from '@grafana/runtime';
+import { TextLink } from '@grafana/ui';
+
+import { useBackendAddress } from 'hooks/useBackendAddress';
+import { useCanWriteSM } from 'hooks/useDSPermission';
+import { useMeta } from 'hooks/useMeta';
+import { LinkedDatasourceView } from 'components/LinkedDatasourceView';
+
+import { Preformatted } from '../../../components/Preformatted';
+import { ConfigContent } from '../ConfigContent';
+
+export function GeneralTab() {
+  const meta = useMeta();
+  // This may be false in play.grafana.net
+  const isSignedIn = config.bootData.user?.isSignedIn ?? false;
+  const canWriteSM = useCanWriteSM();
+  const [backendAddress, backendAddressDescription] = useBackendAddress(true);
+
+  return (
+    <ConfigContent title="General">
+      <ConfigContent.Section>
+        <h4>Private probes</h4>
+        <p>
+          In addition to the public probes run by Grafana Labs, you can also{' '}
+          <TextLink
+            href="https://grafana.com/docs/grafana-cloud/testing/synthetic-monitoring/set-up/set-up-private-probes/"
+            external
+          >
+            install private probes
+          </TextLink>
+          . These are only accessible to you and only write data to your Grafana Cloud account. Private probes are
+          instances of the open source Grafana{' '}
+          <TextLink icon="github" href="https://www.github.com/grafana/synthetic-monitoring-agent">
+            Synthetic Monitoring Agent
+          </TextLink>
+          .
+        </p>
+
+        <h5>Backend address</h5>
```

Comment:
Should be a h4 (well, `<Text element="h4" variant="h5">`.

Reply:
It's sectioned as a part of private probes. I'll just change it to a `h4`